index.html

```diff
@@ -1,11 +1,168 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>HTML Homework 3</title>
-</head>
-<body>
-
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway:wght@700&family=Roboto:ital,wght@0,400;0,500;0,700;0,900;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="./node_modules/modern-normalize/modern-normalize.css"
+    />
+    <link rel="stylesheet" href="css/main.css" />
+
+    <title>WebStudio</title>
+  </head>
+
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <nav class="nav">
+            <a class="logo" href="">
+              <span class="logo__web">Web</span
+              ><span class="logo__studio studio--black">Studio</span>
+            </a>
+            <ul class="nav__list">
+              <li class="nav__item"><a href="" class="nav__link">Студія</a></li>
+              <li class="nav__item">
+                <a href="" class="nav__link">Портфоліо</a>
+              </li>
+              <li class="nav__item">
+                <a href="" class="nav__link">Контакти</a>
+              </li>
+            </ul>
+          </nav>
+          <div class="contacts">
+            <a href="" class="contacts__link">info@devstudio.com</a>
+            <a href="" class="contacts__link">+38 096 111 11 11</a>
+          </div>
+        </div>
+      </div>
+    </header>
+    <main class="main">
+      <div class="page__hero hero">
+        <h1 class="hero__title">Ефективні рішення для вашого бізнесу</h1>
+        <button class="button hero__button">Замовити послугу</button>
+      </div>
+
+      <section class="page__section features__section--pd-btm-0">
```

Comment:
Better

```suggestion
      <section class="page__section features features--pd-btm-0">
```
or
```suggestion
      <section class="page__section section section--pd-btm-0">
```

wherever you use the internal rules for the section (i.e. `padding: 94px 0 94px`), you may move this modificator there as this concerns `page section` in a specific state